R/pkg/inst/profile/shell.R

```diff
@@ -33,7 +33,7 @@
   sc <- SparkR:::callJStatic("org.apache.spark.sql.api.r.SQLUtils", "getJavaSparkContext", spark)
   assign("sc", sc, envir = .GlobalEnv)
   sparkVer <- SparkR:::callJMethod(sc, "version")
-  cat("\n Welcome to")
+  cat("\nWelcome to")
   cat("\n")
   cat("    ____              __", "\n")
   cat("   / __/__  ___ _____/ /__", "\n")
```

Comment:
I think @srowen meant we should have three spaces for this whole logo like:

```
      ____              __
     / __/__  ___ _____/ /__
    _\ \/ _ \/ _ `/ __/  '_/
   /__ / .__/\_,_/_/ /_/\_\
      /_/
```

Can you post the before/after results in the PR description?

Reply:
Updated.